### DISCUSSION TITLE
Help check whether the log is normal. There is no reward for 10 days

### DISCUSSION THREAD

**Discussion by fiser321** [Chia-Network/chia-blockchain#12577] [Support]
![image](https://user-images.githubusercontent.com/8924920/180661378-85913358-6998-408f-b062-fa7ab1a54c84.png)

![image](https://user-images.githubusercontent.com/8924920/180661343-ec5e602e-236d-4ea3-a20d-020ba2e2ec2d.png)
![image](https://user-images.githubusercontent.com/8924920/180661360-2777bf7b-673c-40ae-99e9-c13ed00b5439.png)

help me

**Comment by fiser321** [comment]
I  restart full_node  then   I feel my node is not good node please help me

**Comment by loppefaaret** [comment]
1PiB is close to 0.005% of the network, with less than 5000 blocks a day, you are less than 3x over estimation - it can get alot worse.
from the logs provided, i cant see any glaring issue pop out that would have a negative impact on your farm
